data-fundamentals/README.md

```diff
@@ -0,0 +1,78 @@
+# Data and DataOps fundamentals
+
+Most projects involve some type of data storage, data processing and data ops. For these projects, as with all projects, we follow the general guidelines laid out in other sections around security, testing, observability, CI/CD etc.
+
+## Goal
+
+This goal of this section is to help guide on some of the specifics around how to apply these fundamentals to data projects or data portions of a project.
+
+## Contents
+
+- [Data Tiering (Data Quality)](#data-tiering-data-quality)
+- [Data Validation](#data-validation)
+- [Idempotent Data Pipelines](#idempotent-data-pipelines)
+- [Testing](#testing)
+- [CI/CD, Source Control and Code Reviews](#cicd-source-control-and-code-reviews)
+- [Security and Configuration](#security-and-configuration)
+- [Observability](#observability)
+- [End to End and Azure Technology Samples](#end-to-end-and-azure-technology-samples)
+
+## Data Tiering (Data Quality)
+
+Develop a common understanding of the quality of your datasets so that everyone understands the quality of the data, and expected use cases and limitations.
+
+A common data quality model is `Bronze`, `Silver`, `Gold`
+
+- **Bronze:** This is a landing area for your raw datasets with no to minimal data transformations applied, and therefore are optimized for writes / ingestion. Treat these datasets as an immutable, append only store.
+- **Silver:** These are cleansed, semi-processed datasets. These conform to a known schema and predefined data invariants and might have further data augmentation applied. These are typically used by Data Scientists.
+- **Gold:** These are highly processed, highly read-optimized datasets primarily for consumption of business users. Typically, these are structured in your standard Fact and Dimension tables.
+
+Divide your data lake into three major areas containing your Bronze, Silver and Gold datasets.
+
+> Note: additional storage areas for malformed data, intermediate (sandbox) data, and libraries/packages/binaries are also useful when designing your storage organization
+
+## Data Validation
+
+Validate data early in your pipeline.
```

Comment:
Nit: These "rule of thumb" sentences under other headings don't have punctuation. The "rule of thumb" of consistency says they should be all the same. Uniqueness is sadly frowned upon here. Conformity is the way. Resistance is futile.